packages/baby-jubjub/src/baby-jubjub.ts

```diff
@@ -102,11 +102,11 @@ export function packPoint(unpackedPoint: Point<bigint>): bigint {
         buffer[31] |= 0x80
     }
 
-    return bufferToBigint(buffer)
+    return leBufferToBigint(buffer)
 }
 
 export function unpackPoint(packedPoint: bigint): Point<bigint> | null {
```

Comment:
You can decide on the right best-practices for your library, but I'd suggest that functions like this have more explicit documentation which can explain what a packed point is, and what assumptions users should or shouldn't make about it.  I don't know if this exists, but is there a standard definition of what point packing means, or a compatibility target which could be pointed to here?

Reply:
I don't know if there's a standard but we can definitely look for it!

Documenting these libraries is part of a separate task: https://github.com/privacy-scaling-explorations/zk-kit/issues/142

We definitely need to add more documentation and tests too. The current version is more like an alpha. The goal of this [milestone](https://github.com/orgs/privacy-scaling-explorations/projects/2/views/4) is to release `1.0.0-beta`, to be ready for being audited by the internal PSE team.